libs/service/cennznet/src/fetchProposalInfo.ts

```diff
@@ -62,6 +62,7 @@ export const fetchProposalInfo = async (
 				method: "setCode",
 				args: { code: "<omitted>" },
 			};
+		else throw error;
```

Comment:
terminate early is better

```
// Catch error creating type for `system.setCode`
if (!error?.message.includes("[123, 34]")) throw error
call = {
	section: "system",
	method: "setCode",
	args: { code: "<omitted>" },
};
```

Reply:
I was thinking there may be more cases we want to catch in the future, but we can change it then